blockchain/db/postgres/network_db.py

```diff
@@ -130,8 +130,8 @@ def get_unregistered_nodes(limit=None):
     # TODO: change to (or lost_conn_attempt_ts < 7 days ago and last_activity < 7 days ago) (5 minutes right now)
     #       should possibly be based upon total unregistered/non-connected nodes and how often this is executed
     # TODO: base time interval based on number of attempts (fibonacci series?)
-    query += """ WHERE last_conn_attempt_ts IS NULL OR last_conn_attempt_ts < NOW() - INTERVAL '5 minutes' """
-    query += """ AND start_conn_ts IS NULL AND last_activity_ts < NOW() - INTERVAL '5 minutes' """
+    query += """ WHERE last_conn_attempt_ts IS NULL OR last_conn_attempt_ts < NOW() - INTERVAL '7 day' """
```

Comment:
I believe that this should work, but can you change it to "days" (plural) for readabitily?